gtfs-realtime/spec/en/reference.md

```diff
@@ -20,13 +20,27 @@ Version 1.0 of the feed specification is discussed and documented on this site.
             *   [TripDescriptor](#message-tripdescriptor)
                 *   [ScheduleRelationship](#enum-schedulerelationship-1)
             *   [VehicleDescriptor](#message-vehicledescriptor)
+                *   [CarriageDescriptor](#message-carriagedescriptor)
+                    *   [OccupancyStatus](#enum-occupancystatus)
+                    *   [WheelchairAccessible](#enum-wheelchairaccessible)
+                    *   [ToiletFacilities](#enum-toiletfacilities)
+                    *   [WifiAvailability](#enum-wifiavailability)
+                    *   [AirConditioning](#enum-airconditioning)
+                    *   [BicyclesAllowed](#enum-bicyclesallowed)
             *   [StopTimeUpdate](#message-stoptimeupdate)
                 *   [StopTimeEvent](#message-stoptimeevent)
                 *   [ScheduleRelationship](#enum-schedulerelationship)
         *   [VehiclePosition](#message-vehicleposition)
             *   [TripDescriptor](#message-tripdescriptor)
                 *   [ScheduleRelationship](#enum-schedulerelationship-1)
             *   [VehicleDescriptor](#message-vehicledescriptor)
+                *   [CarriageDescriptor](#message-carriagedescriptor)
+                    *   [OccupancyStatus](#enum-occupancystatus)
```

Comment:
There was a design decision made when originally adding `OccupancyStatus` that `VehicleDescriptor` should only contain immutable information, while `VehiclePosition` should contain mutable statuses (discussion starting at https://groups.google.com/d/msg/gtfs-realtime/_HtNTGp5LxM/86U3GGxDbngJ).  

This proposal breaks from that design decision by putting `OccupancyStatus` as a child element of `VehicleDescriptor->CarriageDescriptor`.  You could also argue that some of the other elements are mutable as well (e.g., bathroom closed for cleaning/maintenance, bikes allowed only at certain times of the day).  

At the time of the `OccupancyStatus` proposal, I actually argued for including `OccupancyStatus` in `VehicleDescriptor` instead of `VehiclePosition`, but was overruled :).  I bring this up for discussion - please discuss :).